### DISCUSSION TITLE
Problem Converting To HLS (request Internal IP)

### DISCUSSION THREAD

**Discussion by 01us01** [bluenviron/mediamtx#768] [Questions]
hi

I have a problem converting to HLS.

When converting HLS, not receiving a response due to internal IP request inside the container.

RTSP address operates normally.

but In the process of converting to HLS through rtsp-simple-server, there is no response because it was requested as an internal IP.
 
Is there a setting to change?

| Request IP 
- aaa.aaa.aaa.aaa:562

| docker logs 
- [rtsp source] ERR: dial tcp 192.168.0.14:554: i/o timeout

| Setup Environment

LTE modem - router(iptime) - device(cctv) 

LTE modem is modem
- port forwarding 562

router(iptime)
- port forwarding 562

add path ( source : rtsp://[camname]:[password]@@[LTE external IP]:562/trackID=1 )

ps. General RTSP works without problems. (ex rtsp://wowzaec2demo.streamlock.net/vod/mp4:BigBuckBunny_115k.mp4)

**Comment by aler9** [comment]
Hello, please post server logs.

[a comment by an automated account is not shown]
